Dockerfile-pachd-skaffold

```diff
@@ -0,0 +1,25 @@
+# syntax = docker/dockerfile:experimental
```

Comment:
Question on the file name: should this just replace the `Dockerfile.pachd` file that's already in our repo? Or is the goal to have both? Would it maybe make sense to put this in `etc/skaffold` or some such? I only ask because the name struck me as long, but maybe it makes sense.

Reply:
Yeah, eventually it will replace the main docker file, I just want to scope out replacing our building container for the purpose of this pr (keep it scoped tightly to skaffold)